terralab/commands/quotas_commands.py

```diff
@@ -0,0 +1,31 @@
+# commands/quotas_commands.py
+
+import logging
+
+import click
+
+from terralab.log import indented
+from terralab.logic import quotas_logic
+from terralab.utils import handle_api_exceptions
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.group()
+def quotas():
+    """Commands to get information about user quotas"""
+
+
+@quotas.command()
+@click.argument("pipeline_name")
+@handle_api_exceptions
+def get_info(pipeline_name: str):
```

Comment:
i dont love this name. halp please :)

Reply:
do we think we're going to have other quota-related commands? what if it's just `terralab quota PIPELINE_NAME` ? or if we prefer verbs (I don't know if I do?) it could be `terralab get-quota PIPELINE_NAME`